web/beacon-app/src/features/onboarding/SetupComplete/OnboardingCompletePageLayout.tsx

```diff
@@ -0,0 +1,16 @@
+import { Route } from 'react-router-dom';
+
+import { routes } from '@/application';
+import MainLayout from '@/components/layout/MainLayout';
+
+import OnboardingCompletePage from './OnboardingCompletePage';
+
+export default function OnboardingCompletePageLayout() {
+  return (
+    <>
+      <Route path={routes.complete} element={<MainLayout />}>
+        <Route path={routes.complete} element={<OnboardingCompletePage />} />
+      </Route>
+    </>
+  );
+}
```

Comment:
i think for now the route is defined in the `application/routes` folder.

Reply:
I created a separate route to show that the onboarding/complete instead of directing the user back to onboarding/setup. I thought it would limit confusion. Also, we may want to display different info on the complete page in the future.